server/src/main/java/org/elasticsearch/search/profile/aggregation/ProfilingAggregator.java

```diff
@@ -124,17 +124,6 @@ public void preCollection() throws IOException {
         profiler.pollLastElement();
     }
 
-    @Override
-    public void postCollection() throws IOException {
```

Comment:
This is kind of an unfortunate loss here. Now we'll end up having `post_collection` in the `build_aggregation` debug phase. I'm wondering if I can have aggregations that do "interesting" things opt in to make their own timers or something.